packages/database/test/order_by.test.ts

```diff
@@ -363,9 +363,7 @@ describe('.orderBy tests', function() {
     expect(addedPrevNames).to.deep.equal(expectedPrevNames);
   });
 
-  it('Removing default listener removes non-default listener that loads all data', function(
-    done
-  ) {
+  it('Removing default listener removes non-default listener that loads all data', function(done) {
```

Comment:
So when I run "yarn prepush", it undoes these changes...  can you run "yarn" from the root of the repo to make sure you have the correct prettier installed and then run "yarn prepush" to see if it creates a commit that undoes these changes for you too?